vendor/izinto/izi-io-4840.yaml

```diff
@@ -0,0 +1,107 @@
+name: izi-io 4840
+description: Multi-I/O Edge Computing Controller with LoRaWAN. The top-hat rail mounted device features 8 analog / RTD / digital inputs, 4 digital outputs as well as industry-standard bus interfaces such as CAN and Modbus.
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '1.1'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '1.0'
+    numeric: 1
+    # Corresponding hardware versions (optional)
+    hardwareVersions:
+      - '1.1'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: izi-io-4840-profile-eu868
+        #lorawanCertified: true
+        ##codec: izi-io-4840-codec
+
+# Sensors that this device features (optional)
+# Valid values are: accelerometer, altitude, auxiliary, barometer, battery, button, co2, distance, dust, gps, gyroscope,
+# humidity, light, link, magnetometer, moisture, ph, pir, proximity, rssi, snr, sound, temperature, tvoc, velocity,
+# vibration, water, wind direction and wind speed.
+sensors:
+  - link
+  - temperature
+
+# Dimensions in mm (optional)
+# Use width, height, length and/or diameter
+dimensions:
+  width: 22.5
+  length: 119
+  height: 101
+
+# Weight in grams (optional)
+weight: 160
+
+### below: TBD
```

Comment:
What is TBD? It seems pretty good?

Reply:
Oh, sorry. That was a marker from an earlier version. All missing information had already been filled in when I did the pull request.

Reply:
I have now deleted the line.